contracts/lib/zeppelin/math/SafeMath.sol

```diff
@@ -1,55 +1,69 @@
-// See https://github.com/OpenZeppelin/openzeppelin-solidity/blob/40b5594f52fce22f6c9190e8e45ccb3cab624783/contracts/math/SafeMath.sol
-// Adapted to use pragma ^0.4.18, use `require()`, and satisfy our linter rules
+// See https://github.com/OpenZeppelin/openzeppelin-solidity/blob/d51e38758e1d985661534534d5c61e27bece5042/contracts/math/SafeMath.sol
+// Adapted to use pragma ^0.4.18 and satisfy our linter rules
 
 pragma solidity ^0.4.18;
 
 
 /**
  * @title SafeMath
- * @dev Math operations with safety checks that throw on error
+ * @dev Math operations with safety checks that revert on error
  */
 library SafeMath {
 
     /**
-    * @dev Multiplies two numbers, throws on overflow.
+    * @dev Multiplies two numbers, reverts on overflow.
     */
-    function mul(uint256 a, uint256 b) internal pure returns (uint256 c) {
-        // Gas optimization: this is cheaper than asserting 'a' not being zero, but the
+    function mul(uint256 _a, uint256 _b) internal pure returns (uint256) {
+        // Gas optimization: this is cheaper than requiring 'a' not being zero, but the
         // benefit is lost if 'b' is also tested.
         // See: https://github.com/OpenZeppelin/openzeppelin-solidity/pull/522
-        if (a == 0) {
+        if (_a == 0) {
             return 0;
         }
 
-        c = a * b;
-        require(c / a == b);
+        uint256 c = _a * _b;
+        require(c / _a == _b);
```

Comment:
Are we checking anything else than overflow here? If it's just overflow, we could do:
```
        uint256 c = _a * _b;
        if (_a > 1) { 
            if(_b > 1) {
                require(c > _a);
                require(c > _b);
            }
        }
        return c;
```
Which is another small gas gain.